packages/paste-website/src/component-examples/ComboboxExamples.ts

```diff
@@ -83,7 +84,7 @@ const OptionTemplateCombobox = () => {
             </Text>
           </MediaBody>
           <MediaFigure spacing="space20">
-            <LinkExternalIcon decorative={false} title="external" />
+            <CalendarIcon decorative={true} />
```

Comment:
issue: this new CalendarIcon isn't showing up in the example; it still has LinkExternalIcon
<img width="719" alt="image" src="https://user-images.githubusercontent.com/77117846/194104535-fb8066bc-175c-4414-b16a-b4cfcdcdbe51.png">